src/me.rs

```diff
@@ -138,6 +126,27 @@ pub fn motion_estimation(
   }
 }
 
+fn full_search(x_lo: isize, x_hi: isize, y_lo: isize, y_hi: isize, blk_h: usize, blk_w: usize,
+    p_org: &Plane, p_ref: &Plane, best_mv: &mut MotionVector, lowest_sad: &mut u32,
+    po: &PlaneOffset, step: usize) {
+  for y in (y_lo..y_hi).step_by(step) {
+    for x in (x_lo..x_hi).step_by(step) {
+      let plane_org = p_org.slice(po);
+      let plane_ref = p_ref.slice(&PlaneOffset { x, y });
+
+      let sad = get_sad(&plane_org, &plane_ref, blk_h, blk_w);
+
+      if sad < *lowest_sad {
+        *lowest_sad = sad;
+        *best_mv = MotionVector {
+          row: 8 * (y as i16 - po.y as i16),
```

Comment:
Would "8 * " still  correct for estimate_motion_ss4() and estimate_motion_ss2() ?
Shouldn't they "32 * " and "16 * " for *_ss4() and _ss2()?

Reply:
Good point. Note that the MVs are scaled by 4x and 2x and at the end of  _ss4() and _ss2() functions, so using 8x here should be correct